docs/spec_draft.md

```diff
@@ -142,9 +142,9 @@ The specification of an op comprises of the following components (in the order
 described below)
 
   * **Semantics** Semantics of the operation.
-  * **Operands** Meaning of operand(s) and their type(s).
-  * **Results** Meaning of the result(s) and the type(s).
-  * **Constraints** Constraints on the operand(s) and the result(s).
+  * **Inputs** Meaning of input(s) and their type(s).
+  * **Outputs** Meaning of the outputs(s) and the type(s).
```

Comment:
Slight typo, here and below:
`outputs(s)`

Reply:
Nice catch!